src/openrct2/world/TileElement.h

```diff
@@ -350,7 +350,11 @@ struct TrackElement : TileElementBase
         struct
         {
             uint8_t Sequence;
-            uint8_t ColourScheme;
+            union
+            {
+                uint8_t ColourScheme;
+                uint8_t SeatRotation;
+            };
```

Comment:
This change is unnecessary and hides the fact that these share bits within a field. You can add a comment detailing which bits are what if you like.